docs/src/troubleshooting.md

```diff
@@ -800,3 +800,68 @@ API. Please check your networking.
 Another possible cause is when you have sidecar injection configured. Sidecars
 such as Istio may make the network temporarily unavailable during startup. If
 you have sidecar injection enabled, retry with injection disabled.
+
+### Bootstrap Job Stuck in running status due to Linkerd sidecar
+
+If your Cluster's initialization job is stuck in the `Running` status, it may be due to the Linkerd sidecar remaining active after the job has completed. Linkerd's proxy `/shutdown` endpoint is **disabled by default**, which prevents the sidecar from gracefully terminating, leaving the job in a stalled state.
+
+When the CNPG operator completes its `import` or `bootstrap` task, it attempts to shut down the sidecars to complete the job. However, without the `/shutdown` endpoint being enabled, Linkerd's proxy remains active, causing the job to never fully exit.
+
+#### Debugging the Issue
+
+To diagnose this issue, you can:
+
+1. **Port-forward** to the `-join1` pod to access the Linkerd proxy.
```

Comment:
the format includes a `-join-1` there's a missing dash, also, do you think it's possible to add the same for the initdb, which have the same problem

Reply:
default              cluster-example-1-initdb-rlsrs                                  1/2     NotReady   0          2m30s
that's the sample I got